src/main/java/htsjdk/samtools/cram/compression/fqzcomp/FQZCompDecode.java

```diff
@@ -0,0 +1,166 @@
+package htsjdk.samtools.cram.compression.fqzcomp;
+
+import htsjdk.samtools.cram.CRAMException;
+import htsjdk.samtools.cram.compression.CompressionUtils;
+import htsjdk.samtools.cram.compression.range.RangeCoder;
+
+import java.nio.ByteBuffer;
+
+public class FQZCompDecode {
```

Comment:
It would be good if each decompressor had a comment describing it's reason for existence, like what sort of data it is good at compressing.  For the ones that were implemented off the C or Javascript a link to that code might be useful for future reference.

Reply:
Done.